src/client/ui/panels/HierarchyPanelContainer.js

```diff
@@ -137,9 +138,13 @@ class HierarchyPanelContainer extends Component {
     return (
       <div
         className={classNames("node", {
-          "is-active": this.props.editor.selected && node.object.id === this.props.editor.selected.id
+          "is-active": this.props.editor.selected && node.object.id === this.props.editor.selected.id,
+          conflict: node.object.missing,
+          "error-root": node.object.isMissingRoot ? node.object.missing : false,
+          disabled: node.object.missing && !node.object.isMissingRoot
         })}
-        onMouseUp={e => this.onMouseUpNode(e, node)}
+        onMouseUp={node.object.missing ? undefined : e => this.onMouseUpNode(e, node)}
+        onMouseDown={node.object.missing ? e => e.stopPropagation() : undefined}
```

Comment:
Hmm, would `null` work here instead of `undefined`? I think we prefer using `null` in code.